ios/UARCTGimbalAdapter/UAGimbalAdapter.m

```diff
@@ -113,11 +117,12 @@ - (void)updateDeviceAttributes {
 
 - (void)stop {
     [Gimbal stop];
+    [[NSUserDefaults standardUserDefaults] setBool:NO forKey:GimbalAdapterStarted];
     UA_LDEBUG(@"Stopped Gimbal Adapter.");
 }
 
 - (BOOL)isStarted {
-    return [Gimbal isStarted];
+    return [[NSUserDefaults standardUserDefaults] boolForKey:GimbalAdapterStarted] && [Gimbal isStarted];
```

Comment:
what are we gaining by adding this layer of NSUserDefaults logic on top of the Gimbal isStarted call? is the method not reliable?

Reply:
Mostly to match Android. Android we track our own start/stop instead of relying on if Gimbal is started if the app is using the adapter and Gimbal directly. It would be weird to say stop the adapter, but start gimbal, and have the adapter started again.